.github/workflows/development-environment.yml

```diff
@@ -65,8 +65,7 @@ jobs:
 
       - name: Set up development environment (as per docs)
         run: |
-          python -m pip install virtualenv
-          python -m virtualenv .venv
+          python -m venv .venv
```

Comment:
We don't need to install `virtualenv` anymore since Python 3 includes it as a module.